# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1323667979 28800
#      Sun Dec 11 21:32:59 2011 -0800
# Branch stable
# Node ID 8497d89a17f171563ceab1a2d55bb70b86a5095e
# Parent  08569c50df6f08d8297d466d3eba76d613de2fa4
mesh.m: Correct typo in documentation

* mesh.m: Correct typo in documentation

diff --git a/scripts/plot/mesh.m b/scripts/plot/mesh.m
--- a/scripts/plot/mesh.m
+++ b/scripts/plot/mesh.m
@@ -29,17 +29,17 @@
 ## correspond to different @var{x} values and rows of @var{z} correspond
 ## to different @var{y} values.
 ##
 ## The color of the mesh is derived from the @code{colormap}
 ## and the value of @var{z}.  Optionally the color of the mesh can be
 ## specified independent of @var{z}, by adding a fourth matrix, @var{c}.
 ##
 ## The optional return value @var{h} is a graphics handle to the created
-## suface object.
+## surface object.
 ## @seealso{colormap, contour, meshgrid, surf}
 ## @end deftypefn
 
 ## Author: jwe
 
 function h = mesh (varargin)
 
   newplot ();
